infra/terraform/platform-base/variables.tf

```diff
@@ -25,6 +25,10 @@ variable "oidc_provider_url" {}
 
 variable "oidc_client_id" {}
```

Comment:
Is this one still used?

Reply:
Yep it is used by the federated_identity module (which I think sets up the saml provider stuff in AWS) and is the correct value.

Reply:
Yep it is used by the federated_identity module (is using the correct value)